RecoLocalCalo/HGCalRecAlgos/interface/HGCalRecHitSimpleAlgo.h

```diff
@@ -43,8 +44,21 @@ class HGCalRecHitSimpleAlgo : public HGCalRecHitAbsAlgo {
         << "makeRecHit: adcToGeVConstant_ not set before calling this method!";
     }
     
+    DetId baseid = uncalibRH.id();
+    unsigned layer = 0;
+    if( DetId::Hcal == baseid.det() && HcalEndcap == baseid.subdetId() ) {
+      layer =  HcalDetId(baseid).depth() + 40;
```

Comment:
are the 40 and 28 below used anywhere else (or are related to)?
these better be defined as constants with descriptive names